.travis.yml

```diff
@@ -40,10 +40,13 @@ jobs:
     script:
     - sudo luarocks install luacheck
     - make luacheck
-    - mkdir --parents $GOPATH/src/golang.org/x
-      && git clone --depth=1 https://go.googlesource.com/lint $GOPATH/src/golang.org/x/lint
-      && go get golang.org/x/lint/golint
-    - go get github.com/vbatts/git-validation
+    - |
+      mkdir -p $GOPATH/src/golang.org/x/tools
```

Comment:
Maybe just `go get -d golang.org/x/lint/golint` to avoid this `mkdir` + `git clone`?